src/main/java/world/bentobox/bentobox/managers/AddonsManager.java

```diff
@@ -108,7 +108,7 @@ private void loadAddon(@NonNull File f) {
         plugin.getLocalesManager().loadLocalesFromFile(addon.getDescription().getName());
 
         // Fire the load event
-        Bukkit.getPluginManager().callEvent(new AddonEvent().builder().addon(addon).reason(AddonEvent.Reason.LOAD).build());
+        new AddonEvent().builder().addon(addon).reason(AddonEvent.Reason.LOAD).build();
```

Comment:
I would say that AddonEvent#builder() could be a static like it is in other events.
It would look a bit cleaner.

Reply:
Good catch. I'll fix that one too.